OptimoveSDK/Sources/Classes/Optimove.swift

```diff
@@ -357,13 +381,13 @@ public extension Optimove {
     @objc func trackIBeaconProximity(beacon: CLBeacon) {
         Optimobile.trackIBeaconProximity(beacon: beacon)
     }
-    
+
     /// Records a notification open event
     /// - Parameter userInfo - The userInfo dictionary you received in the push notification payload
-    @objc public func trackOpenMetric(userInfo: [AnyHashable: Any]) {
```

Comment:
Does this inherit the class `public` access level by default?

Reply:
The default access level by default is internal. We have to explicitly define it as public here.

Reply:
Hm yes, so have we accidentally made this method internal with this change?

Reply:
Ah yes, that's what I was asking! cool 😎